com.unity.netcode.gameobjects/Tests/Runtime/Metrics/Utility/NetworkVariableComponent.cs

```diff
@@ -1,17 +1,17 @@
 #if MULTIPLAYER_TOOLS
-using System;
+using UnityEngine;
 
 namespace Unity.Netcode.RuntimeTests.Metrics.Utlity
 {
     public class NetworkVariableComponent : NetworkBehaviour
     {
-        public NetworkVariableString MyNetworkVariable { get; } = new NetworkVariableString();
+        public NetworkVariable<int> MyNetworkVariable { get; } = new NetworkVariable<int>();
```

Comment:
this needs a review from @becksebenius-unity 

as far as I can tell, it does not matter which type of NetVar we use here but just wanted to double-check to make sure.

Reply:
Yeah this is fine